plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -93,6 +124,14 @@ type CachedPersistentVolumeInfo struct {
 	corelisters.PersistentVolumeLister
 }
 
+func GetPredicatesOrdering() []string {
```

Comment:
Providing getters and setters is not common in our code, but you can add them if you like. However, getters shouldn't have "Get" in their names in Go conventions. You can rename this to `PredicatesOrdering`.